src/parsers/manifest/dash/js-parser/node_parsers/AdaptationSet.ts

```diff
@@ -94,6 +95,16 @@ function parseAdaptationSetChildren(
           children.inbandEventStreams.push(parseScheme(currentElement));
           break;
 
+        case "Label":
+          const [labelObj, labelWarnings] = parseLabel(currentElement);
```

Comment:
I don't see the `parseLabel` function as necessary here considering that there's anyway checks after that call.
We could just set `labelObj` here to `currentElement.textContent` and then check if the value is null / undefined.
Also from what I see, there's no way a parsing error could arise here and as such, `labelWarnings` is always empty. We could consequently just remove that part here.